icons/helicopter.json

```
@@ -0,0 +1,15 @@
+{
+  "$schema": "../icon.schema.json",
+  "contributors": [
+    "liloudreams"
```

@ericfennis and @jamiemlaw also contributed: 

```suggestion
    "liloudreams",
    "ericfennis",
    "jamiemlaw"
```